{"name": "test_incorrect[incorrect_data2]", "status": "passed", "attachments": [{"name": "log", "source": "4d74260b-0967-4fe6-83a5-dd11b93e7f42-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "incorrect_data", "value": "{'username': '1303845892', 'password': 'nmb_python', 'check': '用户不存在'}"}], "start": 1594723066962, "stop": 1594723069871, "uuid": "e9194c09-91ef-474c-92cc-61a2cc8f898a", "historyId": "9a23aa568774cb1eb1113c9dc5db5449", "testCaseId": "486418aa0944ed0ecb75b40c2b58e369", "fullName": "TestCases.test_login.TestLogin#test_incorrect", "labels": [{"name": "tag", "value": "login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KFH4O7N"}, {"name": "thread", "value": "24464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}